components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -94,12 +94,12 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
         setChangesModalVisible(true);
     }
     return <div>
-        <div className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light group">
+        <div className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light group">
             <div className="pr-3 self-center">
                 <WorkspaceStatusIndicator instance={desc?.latestInstance} />
             </div>
             <div className="flex flex-col w-3/12">
-                <a href={startUrl.toString()}><div className="font-medium text-gray-800 truncate hover:text-blue-600">{ws.id}</div></a>
+                <a href={startUrl.toString()}><div className="font-medium text-gray-800 dark:text-gray-100 truncate hover:text-blue-600">{ws.id}</div></a>
```

Comment:
**suggestion**: Could we use `blue-400` for all blue links on dark theme?

Reply:
@gtsiolis Just on hover or always?

Reply:
@jankeromnes only on hover as in light theme. 🌤️